test/freenet/node/PeerNodeTest.java

```diff
@@ -0,0 +1,191 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
```

Comment:
Hmm

Reply:
Argh, sorry. I will add a commit that removes the template text.